index.html

```diff
@@ -1 +1,61 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
+
+<p>The Email Markup Consortium (EMC) is a community-led group of industry professionals working to improve the email
+experience for everyone. Recipients should be able to easily access the content of every email sent to them, regardless
+of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation
+and sending tools, and email clients to drive this vision forward at every step of the email journey.
+
+<p>Senders, recipients, and email clients share the common goals of improving user experience, accessibility, performance,
+consistency, and reliability. However, currently there is a lack of cooperation in trying to achieve these goals leading
+to inconsistent results. By working together on agreed approaches we can save everyone time, effort, and money while
+delivering better results to the recipients.</p>
+
+<h2>How You Can Help</h2>
+
+<ol>
+  <li>PLACEHOLDER: Share on Twitter, LinkedIn, etc (links with share URLs?)</li>
+  <li>PLACEHOLDER: Join us (How? Link to form, GitHub repo)</li>
+</ol>
+
+<h2>Supporters</h2>
+
+<p>The Email Markup Consortium and its mission are supported by employees of Salesforce, Squarespace, Litmus, Email on Acid, Sparkpost, Constant Contact, ActionRocket, Parcel, and more.</p>
+
+<h2>Contributors</h2>
+
+<ul>
+  <li>
+    <a href="https://twitter.com/AliceLiCode">Alice Li</a>
+  </li>
+  <li>
+    <a href="https://dylanatsmith.com">Dylan Smith</a>
+  </li>
+  <li>
+    <a href="https://ellegetsitdone.com">Elle Wilson</a>
+  </li>
+  <li>
+    <a href="https://hussein-alhammad.com">Hussein Al Hammad</a>
+  </li>
+  <li>
+    <a href="https://twitter.com/M_J_Robbins">Mark Robbins</a>
+  </li>
+  <li>
+    <a href="https://www.megbosh.com">Megan Boshuyzen</a>
+  </li>
+  <li>
+    And many more
+  </li>
+</ul>
+
+<footer>
+  <a href="mailto:consortium@emailmarkup.org">Email</a>
```

Comment:
Would linking to the email like this open us up to spambots? Should we think of using a super simple form?

Reply:
I like the idea of using a form instead for this reason

Reply:
Replaced with a placeholder for now.